src/test/regress/sql/multi_utility_statements.sql

```diff
@@ -159,15 +166,29 @@ RETURNS void AS $$
 	DECLARE c3 CURSOR WITH HOLD FOR SELECT * FROM cursor_me WHERE y = $1;
 $$ LANGUAGE SQL;
 
```

Comment:
-- Test DECLARE CURSOR .. WITH HOLD with parameter

Reply:
this is part of `-- Test DECLARE CURSOR .. WITH HOLD with parameter on non-dist key` above